# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
# SPDX-License-Identifier: BSD-3-Clause-Clear

string fs
string type
uint32 total    # MB
uint32 used     # MB
uint32 avail    # MB
uint32 use_rate
string mount
